cmd/kms-rest/startcmd/start.go

```diff
@@ -704,6 +721,28 @@ func startKmsService(params *kmsRestParameters, srv Server) error {
 		handler)
 }
 
+func startMetrics(srv Server, metricsHost string) {
+	if metricsHost == "" {
+		return
```

Comment:
Can we put a warn (or some other type of logger) to say metrics is not configured ?